ouroboros-network/CHANGELOG.md

```diff
@@ -10,6 +10,14 @@
 
 ### Non-breaking changes
 
+## 0.19.0.2 -- 2025-02-03
+
+## 0.19.0.1 -- 2025-01-15
+
+### Non-breaking changes
+
+- Removed `UnsupportedOperation` from rethrow policy.
```

Comment:
I can't find this commit. Cherrypick from @crocodile-dentist 's branch?

Reply:
I just wanted to sneak a changelog fix :grin:.  This change was included in `main` in edfeb17080 (#5038).